### ISSUE TITLE
Is this abandoned?

### ISSUE THREAD

**Issue by simongray** [arachne-framework/aristotle#8]
Hey Luke,

I am looking to maybe use Apache Jena in Clojure for representing some RDF data and I was wondering whether to go with this or perhaps [igraph-jena](https://github.com/ont-app/igraph-jena) and the various igraph-associated libraries.

This project seems to have stopped development in 2018, so I was wondering whether it's totally abandoned nowadays. I do like all of the implemented functionality of the project, but worry that I will be the only one using this.

**Comment by simongray** [comment]
(the project website also returns 404: http://arachne-framework.org/)

**Comment by simongray** [comment]
Another unresolved issue I have is that I don't know how this project is licenced... :S

**Comment by levand** [comment]
Hey @simongray. So Arachne as a framework is probably defunct -- I aimed too highly and never got it to the point where it was really useful.

This library, Aristotle, I am proud of and do think it's useful. I haven't updated it lately because I haven't been working in the space, but I do think it's reasonably complete and functional. While I don't have time these days to devote to maintenance or track down issues for folks, or add new features, I'd certainly be willing to take a look at PRs.

Not sure if it counts as "abandoned" or not, but that's where it's at.

**Comment by simongray** [comment]
That's great to hear. It would be a shame to let something like this go to waste.

BTW you probably don't remember, but we've actually met briefly in Copenhagen back when I had just started working on E&G and you were still attached to the project. Oh, and you also gave us Danes a couple of rides back when we visited Cognitect during the 2028 Conj. Now I've ended up working with knowledge graphs and some other stuff at the University of Copenhagen, still using Clojure though.
